labelbox_dev/session.py

```diff
@@ -98,34 +94,35 @@ def _http_request(cls,
             response = requests.request(**request)
 
         except requests.exceptions.Timeout as e:
-            raise labelbox.exceptions.TimeoutError(str(e))
+            raise TimeoutError(str(e))
         except requests.exceptions.RequestException as e:
             logger.error("Unknown error: %s", str(e))
-            raise labelbox.exceptions.NetworkError(e)
+            raise NetworkError(e)
         except Exception as e:
-            raise labelbox.exceptions.LabelboxError(
+            raise LabelboxError(
                 "Unknown error during Client.query(): " + str(e), e)
 
         if response.status_code is requests.codes.no_content:
             return
 
         try:
+            print(response.content)
             r_json = response.json()
             print(
                 f"Response text: {response.text} Status code: {response.status_code}"
```

Comment:
i know this isn't from this pr, but we prob want a logger instead of print statements.